pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -171,6 +171,30 @@ class CollectionNamespaceSerializer(serializers.Serializer):
     name = serializers.CharField(source="namespace")
 
 
+class CollectionVersionSignatureSerializer(serializers.ModelSerializer):
+    """
+    A serializer for the signatures on a Collection Version.
+    """
+
+    signature = serializers.SerializerMethodField()
+    signing_service = serializers.SlugRelatedField(
+        slug_field="name",
+        allow_null=True,
+        read_only=True,
+    )
+
+    def get_signature(self, obj) -> str:
```

Comment:
nitpick: with pulp_ansible not using the type hinting I'd leave it out to stay consistent with the rest of the codebase.